_docs/roles/redis.md

```diff
@@ -0,0 +1,28 @@
+# Redis
+
+## Redis Setup
+
+The supporting [redis](https://redis.io) setup for this SSO solution consists of a master and two slave instances.
```

Comment:
Again, Redis won't be specific to SimpleSAMLphp -- describe the role in isolation here.